Sources/swift-package-registry/main.swift

```diff
@@ -0,0 +1,13 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Commands
+
+SwiftPackageRegistryTool.main()
```

Comment:
we should probably transition to @main at this point.

@abertelrud @neonichu wdyt about a PR to move SwiftPM to tools version 5.5 and make use of some of the new capabilities?